From fa013edfefd5dabb55e77f654e7f0c138940545f Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Thu, 13 May 2010 19:27:27 -0400
Subject: [PATCH 245/391] powerpc-fix-trace-clock-header

[PATCH] lttng: powerpc trace-clock.h include <asm/time.h> for get_tbl

Make get_tbl() and get_tb() known to powerpc <asm/trace-clock.h> by
having it include <asm/time.h> (where get_tbl() and get_tb() are defined).

This fixes compiler errors when get_tbl() and get_tb() were referenced
by <asm/trace-clock.h> without being defined.

On the other hand, <asm/processor.h> was needlessly included.  Remove it.

From: srikanth krishnakar <skrishnakar@gmail.com>
Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
---
 arch/powerpc/include/asm/trace-clock.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/powerpc/include/asm/trace-clock.h b/arch/powerpc/include/asm/trace-clock.h
index 9aee572..b0b4e21 100644
--- a/arch/powerpc/include/asm/trace-clock.h
+++ b/arch/powerpc/include/asm/trace-clock.h
@@ -11,7 +11,7 @@
 
 #include <linux/timex.h>
 #include <linux/time.h>
-#include <asm/processor.h>
+#include <asm/time.h>
 
 static inline u32 trace_clock_read32(void)
 {
-- 
1.6.5.2

